torch/distributed/tensor/_sharding_prop.py

```diff
@@ -169,7 +171,13 @@ def _propagate_tensor_meta_non_cached(
         # these operators to be inserted in the fx graph.
         from torch.fx.experimental.proxy_tensor import disable_proxy_modes_tracing
 
-        with FakeTensorMode(), disable_proxy_modes_tracing():
+        fake_mode = detect_fake_mode() or FakeTensorMode()
+        suppress_fresh_symbols_ctx = (
```

Comment:
1) can you add a comment explaining why ignore_fresh_unbacked_symbols() is safe here.

Reply:
"once while figuring out the output sharding"
can the output sharding ever reference those new unbacked?